### PR TITLE
[stable/mattermost-team-edition] fix harcoded init image

### PR DESCRIPTION
**Pull request by sebastien-prudhomme**
Signed-off-by: Sébastien Prud'homme <sebastien.prudhomme@gmail.com>

#### Which issue this PR fixes

Allow installing the chart in environments without internet access, by allowing overriding init image.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/mattermost-team-edition/values.yaml [hunk 1]
```diff
@@ -6,6 +6,11 @@ image:
   tag: 5.5.0
   imagePullPolicy: IfNotPresent
 
+initImage:
```
Comment (cpanato) [inline stable/mattermost-team-edition/values.yaml]: I suggest changing to `initContainerImage`
thanks for this PR :)

**Review by cpanato — CHANGES_REQUESTED** [review]
see my comments above

File: stable/mattermost-team-edition/values.yaml (hunk 1 above)
Comment (sebastien-prudhomme) [reply stable/mattermost-team-edition/values.yaml]: As you wish, see no really a consensus between charts about that variable name

Comment (cpanato) [reply stable/mattermost-team-edition/values.yaml]: some charts use this name, thanks for changing

**Comment by sebastien-prudhomme** [comment]
Thanks everyone, the first time one of my PR was merged as fast :-)

**Pull request merged**